packages/evergreen-table/src/components/TableBody.js

```diff
@@ -0,0 +1,18 @@
+import React, { PureComponent } from 'react'
+import { Pane } from 'evergreen-layers'
+
+export default class TableBody extends PureComponent {
+  static propTypes = {
+    ...Pane.propTypes,
+  }
+
+  static defaultProps = {
+    overflowY: 'scroll',
```

Comment:
This long-going property forces tables look ugly in windows, when number of elements is less than the table height. Why not making it `auto`?